src/main/java/racing/RacingGame.java

```diff
@@ -0,0 +1,54 @@
+package racing;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+
+    private final List<Car> cars;
+    private final RandomStrategy rs;
+
+    public RacingGame(List<Car> cars, RandomStrategy rs) {
+        this.cars = cars;
+        this.rs = rs;
+    }
+
+    public RacingGame(String carNames) {
+        this(carNames.split(","));
+    }
+
+    public RacingGame(String[] carNames) {
+        this(Arrays.stream(carNames)
+                .map(Car::new)
+                .collect(Collectors.toList()));
+    }
+    public RacingGame(List<Car> cars) {
+        this(cars, new RealRandomStrategy());
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public List<Car> getWinners() {
+        Map<Integer, List<Car>> groupByPosition = cars.stream().collect(Collectors.groupingBy(Car::getPosition));
+        int winnerPosition = getWinnerPosition(groupByPosition);
+        return getWinnerSortedByName(groupByPosition, winnerPosition);
+    }
+
+    private static List<Car> getWinnerSortedByName(Map<Integer, List<Car>> groupByPosition, int winnerPosition) {
+        List<Car> winners = groupByPosition.get(winnerPosition);
+        winners.sort(Comparator.comparing(Car::getName));
+        return winners;
+    }
+
+    private static int getWinnerPosition(Map<Integer, List<Car>> groupByPosition) {
+        ArrayList<Integer> positions = new ArrayList<>(groupByPosition.keySet());
+        positions.sort(Collections.reverseOrder());
+        return positions.get(0);
+    }
```

Comment:
메서드를 메서드가 호출되는 순서대로 위치시킨다면 코드를 읽기가 조금 더 수월할 것 같아요. 🙂

```suggestion
    private static int getWinnerPosition(Map<Integer, List<Car>> groupByPosition) {
        ArrayList<Integer> positions = new ArrayList<>(groupByPosition.keySet());
        positions.sort(Collections.reverseOrder());
        return positions.get(0);
    }

    private static List<Car> getWinnerSortedByName(Map<Integer, List<Car>> groupByPosition, int winnerPosition) {
        List<Car> winners = groupByPosition.get(winnerPosition);
        winners.sort(Comparator.comparing(Car::getName));
        return winners;
    }
```

추가적으로 이 메서드들을 `정적 메서드`로 선언하신 이유가 있을까요? 🤔

Reply:
함수 분리하다가 static으로 선언된 것을 못보고 지나쳤네요! 의도한 정적 메서드는 아닙니다.

Reply:
`getWinnerPosition`, `getWinnerSortedByName` 두 메서드가 아직 static method로 보이네요. 
의도하신게 아니라면 제거해주세요. 🙏